tests/test_tgis_backend.py

```diff
@@ -681,6 +680,103 @@ def test_tgis_backend_config_load_prompt_artifacts():
                 tgis_be.load_prompt_artifacts("buz", prompt_id1, source_files[0])
 
 
+def test_tgis_backend_register_model_connection():
```

Comment:
You might be able to consolidate these two tests with `@pytest.mark.parametrize`

Reply:
done.